integration-tests/gradle/pom.xml

```diff
@@ -19,6 +19,28 @@
         <mapstruct.version>1.4.2.Final</mapstruct.version>
     </properties>
 
+    <pluginManagement>
+        <plugins>
+            <!-- https://docs.gradle.com/enterprise/maven-extension/#ignoring_arbitrary_files-->
+            <!-- fixes https://github.com/quarkusio/quarkus/issues/33625 -->
+            <plugin>
+            <groupId>com.gradle</groupId>
+            <artifactId>gradle-enterprise-maven-extension</artifactId>
+            <configuration>
+                <gradleEnterprise>
+                <normalization>
+                    <runtimeClassPath>
+                    <ignoredFiles>
+                        <ignoredFile>META-INF/build.properties</ignoredFile>
```

Comment:
The PR title says you are ignoring `config.yml` but it doesn't look like this is what you're doing? Or is `config.yml` a part of `build.properties`?